.github/workflows/cli.yml

```diff
@@ -67,12 +66,6 @@ jobs:
         cd init1
         ../../packages/cli/dist/bin/index.js init
         cd ../
-    - name: test client basic usage run
-      working-directory: ./myworkspace
-      run: |
-        cd init1
-        ../../packages/cli/dist/bin/index.js run ./examples/pipelines/text-bayes-classification.json
-        cd ../
```

Comment:
Running "./examples/pipelines/text-bayes-classification.json" maybe required?

Reply:
Yes, this will check usage for 'run' and pipline.